src/js/view/utils/skin.js

```diff
@@ -165,6 +165,12 @@ export function handleColorOverrides(playerId, skin) {
             ':not(.jw-state-idle) .jw-controlbar',
             '.jw-flag-audio-player .jw-controlbar'
         ], 'background', config.background, true);
+
+        // For touch devices, we want the height of the controlbar to be 88px so it would have a larger touch target.
+        // But we want the background color to only affect the area up to the time slider, which has a height of 61px.
+        // Using a sharp linear gradient to handle this, so need hard-code the transparent area to 27px (88px - 61px)
+        css(`#${playerId}.jw-flag-touch:not(.jw-flag-audio-player):not(.jw-state-idle) .jw-controlbar`,
+            `{background: linear-gradient(transparent 27px, ${config.background} 27px, ${config.background})}`);
```

Comment:
Since this code is far removed from the change in controlbar that sets the 88px height, please include a github.com link to that change specifically in your comment. Ex:

```js
// For touch devices, we set the height of the controlbar to 88px https://github.com/...```

Is it here? https://github.com/jwplayer/jwplayer/blob/b5da824a4b81b29d66bee9e3722abeb8a557dbf9/src/css/controls/imports/controlbar.less#L39

Reply:
Good idea, thanks!